packages/zoe/src/internal-types.js

```diff
@@ -81,6 +81,7 @@
  * @property {(completion: Completion) => void} exitAllSeats
  * @property {ShutdownWithFailure} failAllSeats
  * @property {() => void} stopAcceptingOffers
+ * @property {(string: string) => boolean} isSnoozed
```

Comment:
"snoozed" to me evokes "pause notifications". When you snooze an alarm, the time keeps ticking but it doesn't make sounds.

This is different in that notifications still come out but no offers are _accepted_. I like metaphors but here it might be better to be perfectly explicit: `isAcceptingOffers` (the inverse). It parallels `stopAcceptingOffers`.

Reply:
good suggestion.